api-lib/postmark.ts

```diff
@@ -165,7 +189,7 @@ async function sendEmail(
       'X-Postmark-Server-Token': POSTMARK_SERVER_TOKEN,
     },
     body: JSON.stringify({
-      From: `${FROM_NAME} <${FROM_EMAIL}>`,
```

Comment:
We want to swap the FROM_NAME as well @sshmm

Reply:
@crabsinger  I changed the parameter to from. I used string instead of isColinks as boolean so that we can reuse it later if another email sender is added